provisioner/template.go

```diff
@@ -135,6 +136,12 @@ func renderTemplate(context *templateContext, file string) (string, error) {
 		"zoneDistributedNodePoolGroups": zoneDistributedNodePoolGroups,
 		"certificateExpiry":             certificateExpiry,
 		"sumQuantities":                 sumQuantities,
+		"awsValidID":                    awsValidID,
+		"karpenterNodePools":            karpenterNodePools,
+		"capacityTypes": func() []string {
+			return []string{"spot", "on-demand"}
```

Comment:
Since it's just a static value, can we avoid the function and put that into the templates?